webbpsf/tests/test_psfgrid.py

```diff
@@ -0,0 +1,195 @@
+import astropy.convolution
+from astropy.io import fits
+import numpy as np
+import os
+import pytest
+
+from .. import gridded_library
+from .. import webbpsf_core
+
+
+# @pytest.mark.skip()
```

Comment:
These tests look good. Do we still need the commented-out `# @pytest.mark.skip()` lines?